package.json

```diff
@@ -125,7 +125,7 @@
 			"firefoxProfile": "./test/web-ext-profile",
 			"chromiumProfile": "./test/web-ext-profile",
 			"startUrl": [
-				"https://github.com/refined-github/refined-github/issues"
+				"https://github.com/orgs/refined-github/dashboard"
```

Comment:
This redirects to `https://github.com/` for me...

Reply:
Right, forgot this only works for people in org, my bad :sweat_smile: